tmt/cli.py

```diff
@@ -86,6 +86,34 @@ class Context(click.Context):
     obj: ContextObject
 
 
+@dataclasses.dataclass
+class CLIInvocation:
```

Comment:
```suggestion
class CliInvocation:
```

The last remaining naming inconsistency.

Reply:
Renamed in 20d22859.

FTR, camel case sucks :)